### PR TITLE
Release 3.0.0-rc.3

### PR DESCRIPTION
**Pull request by blackbaud-conorwright**
Trevor has a PR out too, this can wait on. We'll release with his visual tests once that is merged

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.3 (2018-11-28)
+
+- Fixed section forms scrolling as one rather than two separate sections. [#4](https://github.com/blackbaud/skyux-tabs/pull/4)
```
[the remark this answers is not in the document]
Comment (blackbaud-johnly) [reply CHANGELOG.md]: Is it correct that this change is for the vertical tabs component? When I follow the link https://github.com/blackbaud/skyux-tabs/pull/4, the only change I see is to the styles for the sectioned forms component. So I'd have assumed the change is to the sectioned form component, no the vertical tabs component...

Also, what does "to handle vertical overflow for tabs and content separately" mean? Assuming that this resolves the initial issue cited in the PR (https://github.com/blackbaud/skyux2/issues/1343), can we simplify to something like "to keep tabs stationary when uses scroll through content"?

[the remark this answers is not in the document]
Comment (blackbaud-conorwright) [reply CHANGELOG.md]: @Blackbaud-SteveBrush John is right, this is only for the section form component. Vertical tabs are already supporting separate overflows.

@blackbaud-johnly I don't think that would be accurate enough. it's for both sides so yours expanded would be: "to keep tabs stationary when users scroll through content and keep content stationary when users scroll through tabs"

[the remark this answers is not in the document]
Comment (blackbaud-johnly) [reply CHANGELOG.md]: Thanks, Conor. I missed that nuance when I was looking back to the related PR and issue. Sorry about that. ... And FWIW, I think your revised wording looks good. So I guess the final version should be:
`Fixed sectioned form component to keep tabs stationary when users scroll through content and keep content stationary when users scroll through tabs.`

[the remark this answers is not in the document]
Comment (blackbaud-conorwright) [reply CHANGELOG.md]: @Blackbaud-SteveBrush Updated. This fine?

[the remark this answers is not in the document]
Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: "stationary" Who's writing a letter? :)

**Pull request merged**